test/helpers/kubectl.go

```diff
@@ -654,6 +654,13 @@ func (kub *Kubectl) Apply(filePath string) *CmdRes {
 		fmt.Sprintf("%s apply -f  %s", KubectlCmd, filePath))
 }
 
+// ApplyKubeSystem applies the Kubernetes manifest locate at path filepath to kube-system.
+func (kub *Kubectl) ApplyKubeSystem(filePath string) *CmdRes {
```

Comment:
please use the Apply function, the manifest already have the kube-system.

Reply:
done